tests/Tests/600__ai_apps/650__rhosak/650__rhosak.robot

```diff
@@ -1,124 +1,130 @@
-  Select ${NOTEBOOK_CONS_FILENAME} Tab
-  Sleep  3
-  ${consumer_output} =  Get JupyterLab Code Output In a Given Tab  tab_id_to_read=${cons_tab_id}
-  Check Consumer and Producer Output Equality  producer_text=${producer_output}  consumer_text=${consumer_output}
-  Capture Page Screenshot  consumer_run.png
-  Fix Spawner Status
+    # Spawn a notebook with env variables
+    Switch Window    title:Red Hat OpenShift Data Science Dashboard
+    Wait For RHODS Dashboard To Load
+    Launch JupyterHub Spawner From Dashboard
+    Wait Until Page Contains Element    xpath://input[@name="Standard Data Science"]
+    &{notebook_envs}=    Create Dictionary    KAFKA_BOOTSTRAP_SERVER=${bootstrap_server}    KAFKA_USERNAME=${KAFKA_CLIENT_ID}
+    ...    KAFKA_PASSWORD=${KAFKA_CLIENT_SECRET}    KAFKA_TOPIC=${TOPIC_NAME_TEST}
+    ...    KAFKA_CONSUMER_GROUP=${CONSUMER_GROUP_TEST}
+    Spawn Notebook With Arguments    image=s2i-generic-data-science-notebook    envs=&{notebook_envs}
+    # clone JL notebooks from git and run
+    Clone Git Repository    REPO_URL=${GIT_REPO_NOTEBOOKS}
+    Open Consumer Notebook    dir_path=${NOTEBOOK_DIR_PATH}    filename=${NOTEBOOK_CONS_FILENAME}
+    ${cons_tab_id}=    Get Selected Tab ID
+    Open With JupyterLab Menu    Run    Run All Cells
+    Sleep    1
+    Open Producer Notebook    dir_path=${NOTEBOOK_DIR_PATH}    filename=${NOTEBOOK_PROD_FILENAME}
+    ${prod_tab_id}=    Get Selected Tab ID
+    Open With JupyterLab Menu    Run    Run All Cells
+    Wait Until JupyterLab Code Cell Is Not Active In A Given Tab    tab_id_to_wait=${prod_tab_id}
+    Capture Page Screenshot    cell_not_active.png
+    ${producer_output}=    Get JupyterLab Code Output In A Given Tab    tab_id_to_read=${prod_tab_id}
+    Capture Page Screenshot    producer_run.png
+    Select ${NOTEBOOK CONS FILENAME} Tab
+    Sleep    3
+    ${consumer_output}=    Get JupyterLab Code Output In A Given Tab    tab_id_to_read=${cons_tab_id}
+    Check Consumer And Producer Output Equality    producer_text=${producer_output}    consumer_text=${consumer_output}
+    Capture Page Screenshot    consumer_run.png
+    Fix Spawner Status
+    [Teardown]    Clean Up RHOSAK    stream_to_delete=${STREAM_NAME_TEST}
+    ...    topic_to_delete=${TOPIC_NAME_TEST}
+    ...    sa_clientid_to_delete=${KAFKA_CLIENT_ID}
 
 
 *** Keywords ***
 Kafka Suite Setup
-  Set Library Search Order  SeleniumLibrary
+    [Documentation]    Suite setup keyword for RHOSAK component
+    Set Library Search Order    SeleniumLibrary
 
 Kafka Suite Teardown
-  Close All Browsers
+    [Documentation]    Suite teardown keyword for RHOSAK component
+    Close All Browsers
 
 Kafka Test Setup
-  Open Browser  ${ODH_DASHBOARD_URL}  browser=${BROWSER.NAME}  options=${BROWSER.OPTIONS}
-  Login To RHODS Dashboard  ${TEST_USER.USERNAME}  ${TEST_USER.PASSWORD}  ${TEST_USER.AUTH_TYPE}
-  Wait for RHODS Dashboard to Load
-
+    [Documentation]    Test case setup keyword for RHOSAK component
```

Comment:
Would you mind changing the [Documentation] tags to use the 3rd person?  You will find some examples in the [ODS-CI Robot Framework Style Guide](https://docs.google.com/document/d/11ZJOPI1uq-0Wl6a2V8fkAv_TQhfzp9t_IjXAheaJxmQ/edit?usp=sharing)

Reply:
done